WebService/ComparisonEngineUI/ComparisonEngineUI.Android/ComparisonEngineUI.Android.csproj

```diff
@@ -99,7 +112,7 @@
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
   <ProjectExtensions>
     <VisualStudio>
-      <UserProperties TriggeredFromHotReload="True" XamarinHotReloadDebuggerTimeoutExceptionComparisonEngineUIAndroidHideInfoBar="True" />
+      <UserProperties TriggeredFromHotReload="False" XamarinHotReloadDebuggerTimeoutExceptionComparisonEngineUIAndroidHideInfoBar="True" />
```

Comment:
why change this to false?

Reply:
I didn't change it. I think VSdid. Not sure what's the difference